homework/generate-sequence/generateSequence.hpp

```diff
@@ -1,7 +1,27 @@
+// # Generate Sequence
+
+// Napisz funkcję, która przyjmuje 2 argumenty:
+
+// * `int count`
+// * `int step`
+
+// oraz zwraca `std::vector<int>` posiadający `count` elementów i każdy z nich jest większy od poprzedniego o `step`. Pierwszy ma wynosić tyle, ile `step`.
+
+// ## Przykład użycia
+
+// ```cpp
+// auto result = generateSequence(5, 3);  // result = {3, 6, 9, 12, 15}
+// ```
 #pragma once
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
-}
+    std::vector<int> vec;
```

Comment:
Reserve memory for vector, when you know how many elements will be inserted. 
This will avoid the situation with resizing vector few times (which is always expensive).